src/Commands/Migrate.php

```diff
@@ -51,8 +54,14 @@ public function handle()
 
             event(new MigratingDatabase($tenant));
 
-            // Migrate
-            parent::handle();
+            try {
+                // Migrate
+                parent::handle();
+            } catch (Exception $th) {
+                if (! $this->option('skip-failing')) {
+                    throw $th;
+                }
+            }
 
             event(new DatabaseMigrated($tenant));
```

Comment:
We probably want to move both of these `event()` calls inside the `try` block. That way the events (or specifically the second one) doesn't get fired when `parent::handle()` fails.

Reply:
Yeah, that makes sense